llama_index/node_parser/__init__.py

```diff
@@ -1,6 +1,15 @@
 """Node parsers."""
 
-from llama_index.node_parser.simple import SimpleNodeParser
 from llama_index.node_parser.interface import NodeParser
+from llama_index.node_parser.simple import SimpleNodeParser
+from llama_index.node_parser.postprocessors.metadata_extractors import (
+    MetadataExtractor,
+    MetadataExtractorBase,
+)
```

Comment:
i would expose the top-level factory function, the "MetadataExtractor" class that contains links to the individual metadata extractors, and I'd also expose each metadata extractor in the `__init__`. This gives the user control in whether they want to use our top-level metadata extractor or individual extractors